frontend/src/components/FiltersNotApplicable.scss

```diff
@@ -0,0 +1,18 @@
+// Make sure tooltip label text wraps were we expect.
+.usa-tooltip span {
+    width: 242px;
+    display: block;
+    word-wrap:break-word;
+    white-space: normal;
+ }
+ 
+ // Vertical align for the question mark and tool tip button.
+ .usa-tooltip {
+     vertical-align: middle;
+ }
+ 
+ // Horizontally align the question mark with the tool tip button.
+.usa-tooltip svg {
+     height: 1.3em;
+     width: 1.3em;
+ }
```

Comment:
Is this the right place for this, since it will apply to all tooltips? Nothing here is specific to FiltersNotApplicable

Reply:
I added a class for the component.